app/lib/check_answers_fields.yml

```diff
@@ -39,19 +39,12 @@ sections:
         fields:
         - attribute: partner
```

Comment:
We will need to make sure that this attribute gets hidden if the partner feature flag gets switched off. (I think it's going to be a case of adjusting  `CheckAnswers::ApplicantSection` and in particular `#display_fields`)